### PR TITLE
Create Getting Started 1.md

### PR DESCRIPTION
**Pull request by edendasilva**
I think this is how this works?

### CODE REVIEW

**Comment by edendasilva** [comment]
.

File: Getting Started 1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**Getting Started:**
+
+Some of the tools we use may be new to you. That's ok! 
+
+This handbook will guide you through the basics of getting started with the tools we use such as Basecamp, GitHub, and Zoom.
+
+
+
+<u>Basecamp:</u>
```
Comment (justintanner) [inline Getting Started 1.md:9]: Maybe a `###` instead of `<u></u>`

Comment (edendasilva) [reply Getting Started 1.md:9]: That's what I thought, but it doesn't seem to resolve as anything in Typora
heading

Comment (justintanner) [reply Getting Started 1.md:9]: Oh try with a space afterwards `### Zoom:`

**Pull request merged**